### PR TITLE
upgrade version

### PR DESCRIPTION
**Pull request by yaelballa**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ apply plugin: 'spring-boot'
 apply plugin: 'groovy'
 apply plugin: 'signing'
 
-version = '0.8.1.0'
+version = '0.8.4.0'
 group = 'io.github.fallwizard'
```
Comment (yaelballa) [inline build.gradle:9]: @shlomisut should I change group \ version?

File: build.gradle [hunk 2]
```diff
@@ -132,9 +129,9 @@ uploadArchives {
             // when we're ready to publish to Maven Central, use this repository instead
             // (make sure a project has been created for us first!)
             // see: http://central.sonatype.org/pages/ossrh-guide.html#create-a-ticket-with-sonatype
-            repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-                authentication(userName: project.property("sonatypeUsername"), password: project.property("sonatypePassword"))
-            }
+            //repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
```
Comment (yaelballa) [inline build.gradle:132]: no need to push it to public

File: build.gradle [hunk 3]
```diff
@@ -27,12 +27,11 @@ repositories {
...
 }
 
-
 dependencies {
 
-    ext.dropwizardVersion = '0.8.1'
+    ext.dropwizardVersion = '0.8.4'
     ext.springIoPlatformVersion = '1.1.2.RELEASE'
-    ext.jerseyVersion = '1.19'
+    ext.jerseyVersion = '2.22.1'
```
Comment (shlomisut) [inline build.gradle:34]: I would simply remove this - the jersey version should come from the DW dependency.

File: build.gradle (hunk 1 above)
Comment (shlomisut) [inline build.gradle:9]: nope

File: build.gradle (hunk 2 above)
Comment (shlomisut) [inline build.gradle:132]: but we do need to push it to our artifactory. what I do is make these changes in a separate branch, and execute a build from it (the other branch) once the code chages are merged

**Comment by shlomisut** [comment]
@yaelballa just noticed this is the origin Fallwizard :) please make a PR to our fork, later to this

**Comment by yaelballa** [comment]
By mistake... Sorry
On Nov 1, 2015 3:31 PM, "Shlomi Sutton" notifications@github.com wrote:

> @yaelballa https://github.com/yaelballa just noticed this is the origin
> Fallwizard :) please make a PR to our fork, later to this
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Fallwizard/Fallwizard/pull/11#issuecomment-152825967.

## 

This e-mail, as well as any attached document, may contain material which 
is confidential and privileged and may include trademark, copyright and 
other intellectual property rights that are proprietary to Kenshoo Ltd, 
 its subsidiaries or affiliates ("Kenshoo"). This e-mail and its 
attachments may be read, copied and used only by the addressee for the 
purpose(s) for which it was disclosed herein. If you have received it in 
error, please destroy the message and any attachment, and contact us 
immediately. If you are not the intended recipient, be aware that any 
review, reliance, disclosure, copying, distribution or use of the contents 
of this message without Kenshoo's express permission is strictly prohibited.

File: build.gradle (hunk 3 above)
Comment (jar349) [inline build.gradle:34]: This is the big issue with Fallwizard right now: we explicitly use jersey version 1.19 in our code.  2.x is not backwards-compatible and none of us use Fallwizard any longer, so there's been no burning need to take that code and port it to 2.x.

If you all are still using Fallwizard... and you've already ported it, send me a pull request and I'll accept it.

**Pull request closed without merging**